src/models/pokestop.js

```diff
@@ -6,61 +6,15 @@ const { DataTypes, Model, Op, Sequelize } = require('sequelize');
 const sequelize = require('../services/sequelize.js');
 const WebhookController = require('../services/webhook.js');
 const Cell = require('./cell');
-
-const QuestReward = {
-    Unset: 0,
-    Experience: 1,
-    Item: 2,
-    Stardust: 3,
-    Candy: 4,
-    AvatarClothing: 5,
-    Quest: 6,
-    PokemonEncounter: 7,
-    Pokecoin: 8,
-    Sticker: 11,
-    MegaResource: 12,
-};
-
-const ConditionType = {
-    Unset: 0,
-    PokemonType: 1,
-    PokemonCategory: 2,
-    WeatherBoost: 3,
-    DailyCaptureBonus: 4,
-    DailySpinBonus: 5,
-    WinRaidStatus: 6,
-    RaidLevel: 7,
-    ThrowType: 8,
-    WinGymBattleStatus: 9,
-    SuperEffectiveCharge: 10,
-    Item: 11,
-    UniquePokestop: 12,
-    QuestContext: 13,
-    ThrowTypeInARow: 14,
-    CurveBall: 15,
-    BadgeType: 16,
-    PlayerLevel: 17,
-    WinBattleStatus: 18,
-    NewFriend: 19,
-    DaysInARow: 20,
-    UniquePokemon: 21,
-    NpcCombat: 22,
-    PvpCombat: 23,
-    Location: 24,
-    Distance: 25,
-    PokemonAlignment: 26,
-    InvasionCharacter: 27,
-    Buddy: 28,
-    BuddyInterestingPoi: 29,
-    DailyBuddyAffection: 30,
-    MegaEvoPokemon: 37
-};
+const questReward = require('../../static/data/quest_reward.json')
+const conditionType = require('../../static/data/condition_types.json')
```

Comment:
These should probably use the corresponding values in protos instead. (for now, purified-protos probably, or if you dare, use Furtif's repo directly)